graphql-schemas/schema.yml

```diff
@@ -2004,11 +2004,23 @@ confs:
   fields:
   - { name: role, type: Role_v1, isRequired: true }
 
+- name: CodeComponentGitlabSync_v1
+  fields:
+  - { name: sourceProject, type: CodeComponentGitlabSyncProject_v1, isRequired: true }
+  - { name: destinationProject, type: CodeComponentGitlabSyncProject_v1, isRequired: true }
+
+- name: CodeComponentGitlabSyncProject_v1
+  fields:
+  - { name: name, type: string, isRequired: true }
+  - { name: group, type: string, isRequired: true }
```

Comment:
What is group here? Is this akin to "app-sre" or "service" in GitLab?

Reply:
correct. Within app logic, group/name are utilized to supply a gitlab PID for api requests. The alternative was specifying project ID directly but this seemed easier / more clear when reviewing